Form-Controls/index.html

```diff
@@ -13,16 +13,75 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
+  <form class="form" action="reesults.html" method="GET">
     <!-- write your html here-->
     <!-- try writing out the requirements first-->
-  </form>
+     <div>
+        <label for="name">Name</label>
```

Comment:
Nice, I suggest you double check [README](https://github.com/CodeYourFuture/Module-User-Focused-Data/blob/main/Form-Controls/README.md). It seems you miss some points for example:

- valid name as a text string of two characters or more
- I show which element is focused